.artifact/tree.json

```diff
@@ -51303,99 +51303,99 @@
         },
         "position": {
           "position_0.json": {
-            "hash": "c1d032d78232e432d577566e5d5a33b1",
-            "size": 260
+            "hash": "e69477a7744bccdecb2b080c552f993b",
```

Comment:
[note to self - thanks @nicksanford for explaining]

Artifact is a tool written by Eric, and a library included in rdk, used to read data in google cloud storage and save it in a local filesystem for running tests. Especially for large binary files that you don’t want to put in version control.

The tool allows authorized users to publish artifacts to google cloud storage.

Kim added new pose’s to google cloud storage with the measurements in millimeters.

In order to support multiple versions of a given artifact and not have one person updating a file and break the file for everyone else on earlier versions, artifact keeps checksums of the files, which is what is being updated here.